src/value.rs

```diff
@@ -954,6 +958,19 @@ typed_conversions! {
         BTreeMap<K, V> where (K: Into<IString>, V: Into<IValue>);
 }
 
+#[cfg(feature = "indexmap")]
+typed_conversions! {
```

Comment:
This should not duplicate the existing conversions. The prior call should be non-conditional, and this one should *only* contain the IndexMap conversion.